test/e2e/specs/editor/various/format-library/math.spec.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Format Library - Math', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should insert math format with LaTeX', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await editor.canvas
+			.getByRole( 'button', { name: 'Add default block' } )
+			.click();
+
+		await page.keyboard.type( 'equation: ' );
+
+		await editor.clickBlockToolbarButton( 'More' );
+		await page.getByRole( 'menuitem', { name: 'Math' } ).click();
+
+		expect( await editor.getBlocks() ).toMatchObject( [
+			{
+				name: 'core/paragraph',
+				attributes: {
+					content: 'equation: ',
+				},
+			},
+		] );
+
+		const mathInput = page.locator(
+			'.block-editor-format-toolbar__math-input input'
+		);
```

Comment:
```suggestion
		const mathInput = page.getByRole( 'textbox', {
			name: 'LaTeX math syntax',
		} );
```

Nit: This selector can be made more accessible.

Reply:
Thanks, I'll follow-up with a fix so that this definitely makes RC 1.